modules/ossm-operator-install.adoc

```diff
@@ -83,3 +83,74 @@ $ oc new-project istio-system
 $ oc apply -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.12/deploy/servicemesh-operator.yaml
 ----
 
+= Installing the Operators using OperatorHub Web UI
+
+Jaeger and Kiali operators must be installed before the {ProductShortName} operator. 
+
+== Installing the Jaeger Operator
+You must install the Jaeger Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
+* Access to an OpenShift Container Platform cluster using an account with cluster-admin permissions.
+
+.Procedure
+* Navigate in the web console to the Catalog → OperatorHub page.
+
+* Scroll or type "Jaeger" into the Filter to find the Jaeger Operator.
+
+* Select the Operator. If you pick a Community Operator, you are warned that Red Hat does not certify those Operators. You must acknowledge that warning before continuing. Information about the Operator is displayed.
```

Comment:
We do not want customers selecting the Community operator for production use, we want them to select the one that is provided by Red Hat.

Reply:
Do we want to say something along the lines "make sure you select the supported version of the Service Mesh operator"?

Reply:
Yes, I was trying to figure out the best way to word it.  Probably something like "Select the version of the operator that is provided by Red Hat."